deps/anvil/mock_verifier_contract.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Check if jq is installed
+if ! command -v jq &> /dev/null; then
+    echo "Error: jq is required but not installed. Please install jq first."
+    exit 1
+fi
+
+# Check if curl is installed
+if ! command -v curl &> /dev/null; then
+    echo "Error: curl is required but not installed. Please install curl first."
+    exit 1
+fi
+
+# Default Anvil private key
+PRIVATE_KEY="0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
```

Comment:
nit: pkeys could be read from a file to avoid string duplication

Reply:
The repeated variables are gonna be hard to get rid of but I'm working on consolidating the params in a single file and from that, propagate the config to every service